packages/kit/src/runtime/server/page/render.js

```diff
@@ -145,7 +148,11 @@ export async function render_response({
 	}
 
 	if (options.service_worker) {
-		init += `<script>
+		init += options.amp
+			? `<amp-install-serviceworker
+			src="${options.service_worker}"
```

Comment:
According to their [docs](https://amp.dev/documentation/components/amp-install-serviceworker/#src-(required)), src requires link with `https` protocol, which this option does not provide.

Reply:
CMIW but I think it is fine to use relative path assuming that in production it will be deployed in `https` protocol. Like in this official example page: https://amp.dev/documentation/examples/components/amp-install-serviceworker/
```html
<amp-install-serviceworker
    src="/serviceworker.js"
    data-iframe-src="https://amp.dev/serviceworker.html"
    layout="nodisplay">
</amp-install-serviceworker>
```